kubernetes/schema_container.go

```diff
@@ -191,6 +191,12 @@ func volumeMountFields() map[string]*schema.Schema {
 			Optional:    true,
 			Description: `Path within the volume from which the container's volume should be mounted. Defaults to "" (volume's root).`,
 		},
+		"mount_propagation": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Default:     "None",
```

Comment:
There can only be three precise values here, so let's also add a simple validation.

```suggestion
			Default:     "None",
			ValidateFunc: validation.StringInSlice([]string{"None", "HostToContainer", "Bidirectional"}, false),
```